### DISCUSSION TITLE
Lesson 6: returns wrong entrance_fee decimals when printing (05:45:08)

### DISCUSSION THREAD

**Discussion by emilrueh** [smartcontractkit/full-blockchain-solidity-course-py#1784] [Q&A] [answered]
After changing the DECIMALS and STARTING_PRICE back to 8 and 2+3zeros+8zeros (200000000000) and also fixing the rounding error by adding +1 to the return in FundMe.sol Patrick gets a print of 25000000000000000 where as I get a print of only 2500000 (+1).

It is not throwing an error but not behaving like it's supposed to. Backchecked all value but did not find anything not matching the tutorial.

Any ideas as to where the source for this issue may lay?

**Comment by emilrueh** [comment, marked as answer]
I have solved it myself by backchecking my contract code against the lessons code and adjusting the withdraw function. Then deleting the build>deployments>1337 folder therefore resetting the local chain. Then redeployed and interacted with it again.

**Reply by cromewar** [reply]
Nice catch @emilrueh, if you have any other question let us know.
